components/collective-page/sections/Updates.js

```diff
@@ -171,7 +171,14 @@ class SectionUpdates extends React.PureComponent {
                       </P>
                     </Link>
                     {update.userCanSeeUpdate ? (
-                      <HTMLContent content={update.summary} />
+                      <Container>
+                        <HTMLContent content={update.summary} />
+                        <Link route="update" params={{ collectiveSlug: collective.slug, updateSlug: update.slug }}>
+                          <P fontSize="14px">
+                            <FormattedMessage id="UpdateSection.ReadMore" defaultMessage="Read more" />
```

Comment:
Let's re-use the one suggested above, so that we don't add a new string for translators to work on

```suggestion
                            <FormattedMessage id="ContributeCard.ReadMore" defaultMessage="Read more" />
```

Reply:
Fixed this one. Thanks :+1: